Containerfile.rag

```diff
@@ -0,0 +1,37 @@
+# vim: set filetype=dockerfile
+
+ARG OLS_VERSION=latest
+FROM quay.io/openshift/lightspeed-service-api:${OLS_VERSION}
+
+ARG RAG_INDEX="https://github.com/ilan-pinto/lightspeed-rag-documents/releases/latest/download/local.zip"
+ARG APP_ROOT=/app-root
+USER 0
+
+RUN microdnf install -y unzip wget \
+  && mkdir -p vector-db \
+	&& cd vector-db \
+  && wget $RAG_INDEX \
+	&& unzip local.zip \
+	&& mv local ocp-product-docs \
+	&& rm -f local.zip \
+	&& microdnf remove -y unzip wget
+
+RUN chown -R 1001:0 ${APP_ROOT} && \
+    chmod -R g+rx ${APP_ROOT}
+
+# activate virtualenv with workaround RHEL/CentOS 8+
+ENV BASH_ENV="${APP_ROOT}/venv/bin/activate" \
+    ENV="${APP_ROOT}/venv/bin/activate" \
+    PROMPT_COMMAND=". ${APP_ROOT}/venv/bin/activate" \
+    PATH="${APP_ROOT}/venv/bin:${PATH}"
+
+# default user for Python app
+USER 1001
+
+# Run the application
+EXPOSE 8080
```

Comment:
also inherited from the base image as i understand it.  this one is weird since there is no way to "unexpose" a port.  but it is what it is, as i understand it.

Reply:
same as previous - it's also a form of documentation.

Reply:
same as previous, i wouldn't want this image to become misleading if we changed the port used in the base image.